src/backend/joanie/core/management/commands/generate_certificates.py

```diff
@@ -85,12 +85,12 @@ def handle(self, *args, **options):
 
         filters = {}
         if order_uids:
-            filters.update({"uid__in": order_uids})
+            filters.update({"id__in": order_uids})
```

Comment:
```suggestion
            filters.update({"id__in": order_ids})
```

Reply:
the name of the var is order_uids

Reply:
Yep and from my point of view, `order_ids` is a better name to avoid confusion of reader. (Is `uids` equals to `ids` ?)